internal/provider/resource_service.go

```diff
@@ -770,3 +839,40 @@ func buildServiceConnectInput(data *ServiceResourceModel) ServiceConnectInput {
 
 	return ServiceConnectInput{}
 }
+
+func getMultiRegionConfigFromLatestDeployment(ctx context.Context, latestDeployment getServiceInstanceServiceInstanceLatestDeployment) ([]RegionResourceModel, error) {
```

Comment:
This was discussed with the Railway support: it is the "official" way of getting regional config data